static/css/swizzin.css

```diff
@@ -177,3 +177,7 @@ body {
       column-count: 3;
   }
 }
+
+.text-total {
+	color: orange;
```

Comment:
The darkly theme in bootswatch already includes a color that should be fairly appropriate for "orange": the warning colour:

https://bootswatch.com/darkly/

If you change `text-total` to `text-warn` this will fit much better with the whole layout in the event that I add theme support for other bootswatch themes in the future.

Reply:
I have updated all instances of 'text'total' to use the theme class, FYI it appears to be 'text-warning' not 'text-warn'.